components/AppLayout/AppHeader.js

```diff
@@ -1,66 +1,244 @@
+  Typography,
+} from '@material-ui/core';
+import AccountCircleOutlinedIcon from '@material-ui/icons/AccountCircleOutlined';
+import ExitToAppRoundedIcon from '@material-ui/icons/ExitToAppRounded';
+import InfoIcon from '@material-ui/icons/Info';
+import { NAVBAR_HEIGHT, TABS_HEIGHT } from 'constants/size';
 import { EDITOR_FACEBOOK_GROUP, PROJECT_HACKFOLDR } from 'constants/urls';
-import Link from 'next/link';
-import UserName from './UserName';
+import * as Widgets from './Widgets';
+import desktopLogo from './images/logo-desktop.png';
+import mobileLogo from './images/logo-mobile.png';
 
-function AppHeader() {
-  return (
-    <header className="root">
-      <a className="logo hidden-xs" href="/">
-        <h1>{t`Cofacts`}</h1>
-      </a>
-      <nav className="nav">
-        <Link href="/articles">
-          <a className="nav-item">{t`Collected Messages`}</a>
-        </Link>
-        <Link href="/replies">
-          <a className="nav-item">{t`Replies`}</a>
-        </Link>
-        <a
-          href={EDITOR_FACEBOOK_GROUP}
-          target="_blank"
-          rel="noopener noreferrer"
-          className="nav-item hidden-xs"
-        >
-          {t`Editor forum`}
-        </a>
-        <a
-          href={PROJECT_HACKFOLDR}
-          target="_blank"
-          rel="noopener noreferrer"
-          className="nav-item hidden-xs"
-        >
-          {t`About`}
-        </a>
-      </nav>
-      <UserName />
-      <style jsx>{`
-        .root {
-          display: flex;
-          align-items: center;
-          padding: 0 24px;
-        }
-        .logo {
-          margin-right: 16px;
-        }
-        .nav {
-          margin-right: auto;
-          display: flex;
-        }
-        .nav-item {
-          padding: 8px;
-          border-left: 1px dashed #ccc;
-        }
+const MENU_BUTTON_WIDTH = 48;
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    position: 'sticky',
+    height: NAVBAR_HEIGHT + TABS_HEIGHT,
+    top: 0,
+    zIndex: 10,
+    [theme.breakpoints.up('md')]: {
+      height: NAVBAR_HEIGHT,
+    },
+  },
+  flex: {
+    display: 'flex',
+  },
+  top: {
+    height: NAVBAR_HEIGHT,
+    padding: '0 1rem',
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    background: '#FFFFFF',
+    [theme.breakpoints.up('md')]: {
+      padding: '0 2rem',
+    },
+  },
+  logo: {
+    width: 100,
+    height: 'auto',
+    [theme.breakpoints.up('md')]: {
+      width: 240,
+    },
+  },
+  tabs: {
+    marginRight: 'auto',
+    display: 'flex',
+    justifyContent: 'space-between',
+    height: TABS_HEIGHT,
+    width: `calc(100% - ${MENU_BUTTON_WIDTH}px)`,
+    backgroundColor: theme.palette.secondary[50],
+    [theme.breakpoints.up('md')]: {
+      backgroundColor: 'inherit',
+      height: 'auto',
+      width: 'auto',
+      fontSize: 20,
+      padding: '0 10px',
+    },
+  },
+  tab: {
+    padding: 8,
+    flex: '1',
+    lineHeight: `calc(${TABS_HEIGHT}px - 8px * 2)`,
```

Comment:
If we can ensure all `.tab`s are flex child (with `align-items` being `stretch`), I prefer using flexbox to vertically center the text, i.e. `display: flex` + `align-items: center` here, rather than using `lineHeight` + `calc`. In this way we don't need to keep paddings & height in sync.